src/_P016_IR.ino

```diff
@@ -725,18 +724,12 @@ boolean Plugin_016(uint8_t function, struct EventStruct *event, String& string)
         state.sleep    = -1;
         state.clock    = -1;
 
-        String description = IRAcUtils::resultAcToString(&results);
+        if (loglevelActiveFor(LOG_LEVEL_INFO)) {
+          String description = IRAcUtils::resultAcToString(&results);
 
-        if (!description.isEmpty()) {
-          if (loglevelActiveFor(LOG_LEVEL_INFO)) {
+          if (!description.isEmpty()) {
             // If we got a human-readable description of the message, display it.
-            String log;
-
-            if (log.reserve(10 + description.length())) {
-              log += F("AC State: ");
-              log += description;
-              addLogMove(LOG_LEVEL_INFO, log);
-            }
+            addLogMove(LOG_LEVEL_INFO, strformat(F("AC State: %s"), description));
```

Comment:
This fails compilation as you have to pass the `.c_str()` pointer for strformat to handle in the `%s` parameter.